### ISSUE TITLE
Nano S Plus support and continue updates

### ISSUE THREAD

**Issue by daic11** [LedgerHQ/app-bitcoin-legacy#262]
Hello why did you stop development of Bitcoin Liquid Network in Ledger App?
The last update was 3 years ago. It still not support Nano S Plus and I want to build it for myself.
Do you have any recommendation what I should change to build it for Nano S Plus please?

**Comment by vforgeoux-ledger** [comment]
Hey @daic11 , is Liquid network still alive ?

**Comment by daic11** [comment]
@vforgeoux-ledger yeah, and it's cool beacause it's a 2nd layer of bitcoin but not the different blockchain.
Blockstream (company that develops Liquid) mainly targeting this onto corporate clients.
The coolness of Liquid is privacy, speed and nearly on-chain security benefits because of their interesting consensus algorithm based on their Federation consensus.

So, that's a very cool technology but still need more infrastructure and marketing.
And unfortunately Ledger stopped supporting it. But Nano S still has an old app that is not supported for now.
I also tried to build it by myself on my Ledger Nano S+ but there are difficulties connected with security issues while opening a self-builded app in debugging mode.

So, I asked developers to continue developing it, but the contributor of the new Ledger Bitcoin app has a lot of tasks before Liquid. So, idk when they even will start developing this. And also don't know if they will be.
